std_msgs/Header header
FingerData[] fingers